notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<p>I think this warning is coming from the values in your categorical columns. Not sure what <code>quoted_name</code> is? Maybe look at the initial merges and SQL query you do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I think it has something to do with some of the data having labels (pandas df/series) and some of it having gotten reduced to numpy arrays somewhere, and it sees that these two things look different and thinks maybe you're not using the same kind of data for training / prediction?